pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/util/ConfigManager.java

```diff
@@ -74,29 +77,49 @@ public static synchronized ConfigManager getInstance() {
      * please use {@link #getInstance()}
      */
     public static Config parseConfig() {
-        if (TYPESAFE_CONFIG_FILE != null) {
-            log.info("The config file name was specified. Loading configuration from the local file");
-            return ConfigFactory.parseFile(TYPESAFE_CONFIG_FILE);
+        Optional<Config> configCloud = Optional.empty();
+        if (isSecretManagerConfigurationSpecified()) {
+            log.info("Secret manager configuration found. Trying to load the configuration from the Secret Manager");
+            configCloud = Optional.of(loadFromSecretManager());
+        }
+
+        Optional<Config> configLocal = Optional.empty();
+        if (isLocalConfigurationFileSpecified()) {
+            log.info("The config file name was specified. Trying to load the configuration from the local file");
+            configLocal = Optional.of(ConfigFactory.parseFile(TYPESAFE_CONFIG_FILE));
+        }
+
+        if (configCloud.isEmpty() && configLocal.isEmpty()) {
+            log.error("Can't load configuration. Neither local config not cloud one was specified");
+            throw new DDPException("Can't load configuration. Neither local config not cloud one was specified");
```

Comment:
Suggested update to the message to tell the user exactly what needs to be fixed.

```suggestion
        if (configCloud.isEmpty() && configLocal.isEmpty()) {
            log.error("no configuration was specified. Use properties '{}' to use a local file or '{}' and '{}' to use a cloud secret",
                    TYPESAFE_CONFIG_SYSTEM_VAR,
                    GOOGLE_SECRET_PROJECT,
                    GOOGLE_SECRET_NAME);
            throw new DDPException("no configuration was specified.");
```